### DISCUSSION TITLE
goStrongswanVici

### DISCUSSION THREAD

**Discussion by jhay06** [strongswan/strongswan#1091] [Q&A]
Hi anyone can give me a good example on how to use goStrongswanvici  ? 
https://pkg.go.dev/github.com/bronze1man/gostrongswanvici#section-readme

Thanks

**Comment by tobiasbrunner** [comment]
Isn't there a lengthy example right there? Anyway, I don't know that module. You might want to consider using [govici](https://github.com/strongswan/govici) instead.
